cpp/src/gandiva/function_registry_math_ops.cc

```diff
@@ -62,11 +73,16 @@ namespace gandiva {
 std::vector<NativeFunction> GetMathOpsFunctionRegistry() {
   static std::vector<NativeFunction> math_fn_registry_ = {
       MATH_UNARY_OPS(cbrt, {}), MATH_UNARY_OPS(exp, {}), MATH_UNARY_OPS(log, {}),
-      MATH_UNARY_OPS(log10, {}),
+      MATH_UNARY_OPS(log10, {}), MATH_UNARY_OPS(sqrt, {}),
```

Comment:
There are existing sqrt functions in function_registry_arithmetis.cc.

Reply:
Sorry about that, I think I made a mess with the rebase. I will push an updated commit soon.